payments-ui-core/src/main/java/com/stripe/android/ui/core/CardScanActivity.kt

```diff
@@ -0,0 +1,49 @@
+package com.stripe.android.ui.core
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.stripe.android.PaymentConfiguration
+import com.stripe.android.stripecardscan.cardscan.CardScanSheetResult
+import com.stripe.android.ui.core.databinding.ActivityCardScanBinding
+
+const val CARD_SCAN_PARCELABLE_NAME = "CardScanActivityResult"
+
+class CardScanActivity : AppCompatActivity() {
```

Comment:
we should mark this class internal or at least `@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)`